lib/translations/codegen_loader.g.dart

```diff
@@ -184,7 +190,9 @@ static const Map<String,dynamic> es = {
   "setting_screen": {
     "location_services_disabled": "Los servicios de localización están desactivados.",
     "location_permission_denied": "Los permisos de localización están denegados.",
-    "location_permission_denied_permanently": "Los permisos de localización están permanentemente denegados."
+    "location_permission_denied_permanently": "Los permisos de localización están permanentemente denegados.",
+    "switch_language_description": "Switch the language of the application",
+    "throw_test_exception": "Lanzar una excepción de prueba"
   },
   "langage": {
     "english_langage": "Inglés",
```

Comment:
You need to run flutter pub run easy_localization:generate -S "assets/translations" -O "lib/translations" to update keys